integration-tests/security/src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ org.acme.openapi.security.auth.basic_auth/password=test
 # since the file name has a space, we use the URI representation of this space here to not break the properties file
 # see the RFC3986 for more info https://datatracker.ietf.org/doc/html/rfc3986
 quarkus.openapi-generator.codegen.spec.open_weather_yaml.base-package=org.acme.openapi.weather
+quarkus.openapi-generator.codegen.spec.open_weather_yaml.client-header-factory=none
```

Comment:
I think it would be better to create a new integration test instead of changing this one. With a new integration test, we would cover both scenarios.

Reply:
Would it be ok if i extend existing integration test with 2 new yaml files and use them with new option?
1. client-header-factory=none
2. client-header-factory=org.eclipse.microprofile.rest.client.ext.DefaultClientHeadersFactoryImpl.class

Reply:
Sure. You can extend the existing one.

Reply:
done